README.md

```diff
@@ -49,9 +48,13 @@ Coming soon, will be live by December 6. You can use manually in the mean time p
 You can manually install the library by cloning this repo and creating a Podfile that references your local clone of it like below:
 
 ``` ruby
-  pod 'Amplify', :path => '~/Projects/Amplify/amplify-ios'
-  pod 'AWSPluginsCore', :path => '~/Projects/Amplify/amplify-ios'
-  pod 'CoreMLPredictionsPlugin', :path => '~/Projects/Amplify/amplify-ios'
-  pod 'AmplifyPlugins/AWSPredictionsPlugin', :path => '~/Projects/Amplify/amplify-ios'
+  pod 'Amplify', :path => '~/amplify-ios'
+  pod 'AWSPluginsCore', :path => '~/amplify-ios'
+  pod 'CoreMLPredictionsPlugin', :path => '~/amplify-ios'
+  pod 'AmplifyPlugins/AWSPredictionsPlugin', :path => '~/amplify-ios'
 ```
-You also need to go to your target project by clicking the top level project in Xcode and then clicking under Targets on your project. Then head to Build Phases -> Link Binary with Libraries -> Add Amplify Frameworks and any others you need for the category or categories you would like to use.
```

Comment:
consider putting this back. I also thought we didn't need it but Michael Lab said he had to do this to get it to build.

Reply:
pod's from a local path should be the same as getting it from a repository and shouldn't need more than that, unless a dependent pod being referenced is not published yet, then they'll need additional pods. I added an explicit instruction to use the workspace file.